src/PostgREST/Plan/Types.hs

```diff
@@ -20,6 +21,7 @@ import PostgREST.SchemaCache.Identifiers (FieldName)
 import Protolude
 
 type TransformerProc = Text
+type ToTsVector = Maybe (Maybe Text)
```

Comment:
This is confusing at first glance, why a double Maybe?

Reply:
True, I created a new type to make it make sense.